public/app/features/manage-dashboards/components/ImportDashboardOverview.tsx

```diff
@@ -14,7 +14,7 @@ interface ConnectedProps {
   inputs: DashboardInputs;
   source: DashboardSource;
   meta?: any;
-  folderId: number;
+  folderId: { id: number; title?: string };
```

Comment:
Consider splitting this into 2 different props or change name to folder/folderConfig/folderSettings

Reply:
Or where is title used?

Reply:
I wanted to match the type that the FolderPicker returns

Reply:
...and the FolderPicker should return a `SelectableValue<number>` in the future in my opinion. But that's a bit out of scope for this PR imo.

Reply:
Gotcha, then we should just rename it folder for now?